app/enterprise/2.1.x/deployment/hybrid-mode.md

```diff
@@ -72,12 +72,12 @@ Plane in Hybrid mode.
 ### Kubernetes support
 
 [Kong Enterprise on Kubernetes](/enterprise/{{page.kong_version}}/kong-for-kubernetes/install-on-kubernetes)
-supports Hybrid mode deployments. However, for the 2.1.x Beta release, the nodes
-will not appear in Kong Manager.
+fully supports Hybrid mode deployments.
 
-Kong for Kubernetes Enterprise (K4K8s, or the Kong Ingress Controller with
-{{site.base_gateway}}) does not use a database and is therefore not supported in
-Hybrid mode.
+[Kong for Kubernetes Enterprise]((/enterprise/{{page.kong_version}}/kong-for-kubernetes/install) 
+(K4K8s, or the Kong Ingress Controller with {{site.base_gateway}}) supports Hybrid 
+mode with one limitation: Kong Manager displays configuration in read-only format 
+for DB-less mode.
```

Comment:
This is I guess something we field back to Product, but essentially this distinction is going away; IMO we can just say it fully supports Hybrid Mode (the previous block) and drop this second block entirely.

For Hybrid Mode deployments, Kong Manager supports everything, since there's a DB-backed node somewhere it can read from and write config to. It's kind of in a weird space since we discourage mixing manual and controller-managed configuration, but there are some things that are never controller-managed (e.g. Dev Portal config) where Hybrid Mode has full Manager support, and we expect users to use Manager for them.

For _not_ Hybrid Mode deployments, Kong Manager supports can provide a read-only view of configuration on fully DB-less deployments, but that should be out of scope for the Hybrid Mode docs.